### DISCUSSION TITLE
Require doesnt work

### DISCUSSION THREAD

**Discussion by cheesycod** [mlua-rs/mlua#434] [General]
thread 'lang_lua::test::lua_test' panicked at core/rust.templating/src/lang_lua/mod.rs:174:53:
called `Result::unwrap()` on an `Err` value: RuntimeError("[string \"core/rust.templating/src/lang_lua/mod.rs:162:16\"]:1: module 'io' not found\nstack traceback:\n\t[C]: in ?\n\t[C]: in ?\n\t[string \"core/rust.templating/src/lang_lua/mod.rs:162:16\"]:1: in ?")

    let lua = Lua::new_with(
        LuaStdLib::ALL_SAFE,
        LuaOptions::new().catch_rust_panics(true),
    )?;

When using require

**Comment by cheesycod** [comment]
Ok, this seems to be an mlua 0.10 bug

**Comment by khvzak** [comment]
It's fixed
